src/js/references/components/Detail/ReferenceManage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { match } from "react-router-dom";
+import styled from "styled-components";
+import { BoxGroup, BoxGroupHeader, ContainerNarrow, LoadingPlaceholder, Table } from "../../../base";
+import { Contributors } from "../../../indexes/components/Contributors";
+import { useGetReference } from "../../hooks";
+import { Clone } from "./Clone";
+import { LatestBuild } from "./LatestBuild";
+import RemoteReference from "./Remote";
+import Targets from "./Targets/Targets";
+
+const ReferenceManageTable = styled(Table)`
+    th {
+        width: 180px;
+    }
+
+    tr:not(:first-of-type) td {
+        text-transform: capitalize;
+    }
+`;
+
+type ReferenceManageProps = {
+    /** Match object containing path information */
+    match: match<{ refId: string }>;
+};
+
+/**
+ * Displays the reference manage view showing information about the reference
+ */
+export default function ReferenceManage({ match }: ReferenceManageProps) {
```

Comment:
```suggestion
export default function ReferenceManager({ match }: ReferenceManageProps) {
```
? Manage feels awkward to me